it-and-security/teams/workstations-canary.yml

```
@@ -53,6 +53,7 @@ controls:
     - path: ../lib/macos-remove-old-nudge.sh
     - path: ../lib/windows-remove-fleetd.ps1
     - path: ../lib/windows-turn-off-mdm.ps1
+    - path: ../lib/install_bitdefender_v5.ps1
```

```suggestion
    - path: ../lib/windows-install-bitdefender-v5.ps1
```

Small nit: Updated naming to be consistent w/ other scripts: platform prefix (here `windows`) and dash (`-`) instead of underscores (`_`).

Will the scripts always install Bitdefender `v5` or does it just install the latest? 

Wondering if we can cut `the -v5` from the script name so that we don't have to update it later. When Windows released a new version.